applications/ShallowWaterApplication/custom_utilities/move_shallow_water_particle_utility.h

```diff
@@ -417,7 +417,7 @@ class MoveShallowWaterParticleUtility
     {
         KRATOS_TRY
         ModelPart::NodesContainerType::iterator inodebegin = mrModelPart.NodesBegin();
-        vector<unsigned int> node_partition;
+        std::vector<unsigned int> node_partition;
```

Comment:
if you use OpenMPUtils, you will not need to use the ifdefs (and it will also work without openmp, while your code would break)